docs/anomaly-detection/internet-of-things/process.sql

```diff
@@ -17,20 +17,14 @@ CREATE STREAM throughputs (
   KAFKA_TOPIC='throughputs',
   PARTITIONS = 6);
 
--- Filter throughputs below threshold 1000.0
-CREATE STREAM throughput_insufficient AS
-  SELECT *
-  FROM throughputs
-  WHERE throughput < 1000.0
-  EMIT CHANGES;
-
--- Create new stream where threshold is insufficient and alarm code is not 0
-CREATE STREAM critical_issues_to_investigate AS
+-- Create new stream of critial issues to investigate
+-- where throughputs are below threshold 1000.0 and alarm code is not 0
+CREATE STREAM critical_issues WITH (KAFKA_TOPIC='critical_issues') AS
```

Comment:
Is the renaming of the STREAM and topic intended (ie, does it not break anything?)

Reply:
@mjsax it was intentional (motivation was simply to shorten the stream name so it displays better), and there is nothing else downstream dependent on these names so it's ok.